packages/mail/src/classes/mail-service.js

```diff
@@ -83,6 +83,7 @@ class MailService {
           rule.pattern.test('');
           return rule
         } catch (err) {
+          // FIXME?
```

Comment:
The build currently fails ESLint due to an empty code block.
I'm mystified as to why that happens, because my PR doesn't change anything else in this entire file.